src/OddJobs/Types.hs

```diff
@@ -33,10 +34,10 @@ import Control.Monad.Logger (LogLevel)
 -- myJobsTable :: TableName
 -- myJobsTable = "my_jobs"
 -- @
-type TableName = PGS.Query
+type TableName = PGS.Identifier
```

Comment:
I think the comment on this type should be updated, as it refers to the old underlying `Query` type.  Perhaps it could be phrased in a way that says what it is for, rather than what its type is, so that it can't get out of sync with the code.  Something like "A type alias used for table names. ..."?

Reply:
done